services/motion/builtin/builtin_test.go

```diff
@@ -300,7 +303,10 @@ func TestMoveOnMapPlans(t *testing.T) {
 			ComponentName: base.Named("test-base"),
 			Destination:   goalInSLAMFrame,
 			SlamName:      slam.Named("test_slam"),
-			Extra:         extra,
+			MotionCfg: &motion.MotionConfiguration{
+				PlanDeviationMM: 1,
```

Comment:
Why do we need to add this?

Reply:
```
goalInBaseFrame := spatialmath.NewPose(r3.Vector{X: 1.32 * 1000, Y: 0}, &spatialmath.OrientationVectorDegrees{OZ: 1, Theta: 33})

goalInSLAMFrame := spatialmath.PoseBetweenInverse(motion.SLAMOrientationAdjustment, goalInBaseFrame)
```

the existing goal is within 2.6m so we change it to 1

Reply:
then we should change the existing goal